### DISCUSSION TITLE
Conflicting docs & released descriptions

### DISCUSSION THREAD

**Discussion by zezba9000** [microsoft/WindowsAppSDK#2239] [Q&A] [answered]
I'm wanting to install the "Windows App SDK" but each place I look docs or descriptions are conflicting.

In MS docs it shows 1.0 as stable release
https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/stable-channel#version-10
![image](https://user-images.githubusercontent.com/1857902/157779555-78e7f021-0afe-4626-ad4b-e109fd3ad0e0.png)

In VS 2022 Extension manager it shows as 1.0.0.xxxx but labeled as Preview / Experimental
![image](https://user-images.githubusercontent.com/1857902/157779701-135e17d0-5109-45ec-9a3b-5585e247ced3.png)

On GitHub the version released is only 0.8.7 preview.
![image](https://user-images.githubusercontent.com/1857902/157779783-2280c926-289c-49d4-b045-8c4d20bc5b62.png)

What is what? This is the most confusing thing I've ever seen in any Windows SDK.

**Comment by ghost** [comment, marked as answer]
experimental(v1) version is available in VS Marketplace.
Stable(v1)/Servicing for Older(v0.8.7)  version is available in MS website.

Yes its confusing. MS should do better
